docs/DeveloperGuide.adoc

```diff
@@ -816,22 +816,72 @@ _{More to be added}_
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is `VetterAppointments` and the *Actor* is the `vet`, unless specified otherwise)
+
+
+[discrete]
+=== Use case: Add Vet Technician
+
+*MSS*
+
+1.  Vet requests to add Vet Technician
+2.  VetterAppointments adds the Vet Technician to the system
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The format of command is invalid
++
+[none]
+** 1a1. VetterAppointments shows an error message
++
+Use case ends.
```

Comment:
I think have to change to:  Use case resumes at step 1
Same goes for code below.

Since the the MSS has to end at a user adding the vet technician.
Not really sure though... can change in the future if necessary.